src/model_theory/basic.lean

```diff
@@ -47,19 +47,19 @@ namespace first_order
   type of relations of every natural-number arity. -/
 @[nolint check_univs] -- intended to be used with explicit universe parameters
 structure language :=
-(functions : ℕ → Type u) (relations : ℕ → Type v)
+(functions : ℕ → Type u) (relations : ℕ → Type u)
```

Comment:
Can you add a brief comment to the docstring explaining why you force these into the same universe?